openbb_platform/providers/deribit/openbb_deribit/models/futures_curve.py

```diff
@@ -0,0 +1,186 @@
+"""Deribit Futures Curve Model."""
+
+# pylint: disable=unused-argument
+
+from typing import Any, Optional, Union
+
+from openbb_core.app.model.abstract.error import OpenBBError
+from openbb_core.provider.abstract.fetcher import Fetcher
+from openbb_core.provider.standard_models.futures_curve import (
+    FuturesCurveData,
+    FuturesCurveQueryParams,
+)
+from openbb_core.provider.utils.descriptions import (
+    QUERY_DESCRIPTIONS,
+)
+from openbb_core.provider.utils.errors import EmptyDataError
+from openbb_deribit.utils.helpers import (
+    DERIBIT_FUTURES_CURVE_SYMBOLS,
+    FuturesCurveSymbols,
+)
+from pydantic import Field, field_validator, model_validator
+
+
+class DeribitFuturesCurveQueryParams(FuturesCurveQueryParams):
+    """
+    Deribit Futures Curve Query.
+
+    Source: https://docs.deribit.com/?shell#public-ticker
+    """
+
+    __json_schema_extra__ = {
+        "symbol": {
+            "multiple_items_allowed": False,
+            "choices": DERIBIT_FUTURES_CURVE_SYMBOLS,
+        },
+        "hours_ago": {
+            "multiple_items_allowed": True,
+        },
+    }
+
+    symbol: FuturesCurveSymbols = Field(
+        default="BTC",
+        description=QUERY_DESCRIPTIONS.get("symbol", "")
+        + " Default is 'btc' Supported symbols are: ['btc', 'eth', 'paxg']",
+    )
+    hours_ago: Optional[Union[int, list[int], str]] = Field(
+        default=None,
+        description="Compare the curve with the curve from the specified hours ago. Default is None.",
```

Comment:
What curve?

Reply:
The name of the function is "futures_curve", am I assuming too much in the description?